### PR TITLE
Change teamworkdevices - Add enum type : sipAnalog

### PR DESCRIPTION
**Pull request by AbAgrawal15**
Adding a new deviceType sipAnalog

### CODE REVIEW

**Comment by AbAgrawal15** [comment]
@microsoft-github-policy-service agree [company="microsoft"]

**Comment by AbAgrawal15** [comment]
@microsoft-github-policy-service agree

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 1]
```diff
@@ -1035,6 +1035,24 @@
...
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "3ba6da9a-ff6d-4100-93b4-90324f8fa2d6",
+          "ApiChange": "Enumeration",
+          "ChangedApiName": "teamworkdevices",
+          "ChangeType": "Addition",
+          "Description": "Added the **sipAnalog** enumeration type.",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.GraphSvc.json:1046]: This description should actually be: "Added the `sipAnalog` value to the **teamworkDeviceType" enumeration."

**Comment by Lauragra** [comment]
@AbAgrawal15 , please follow the Get started steps to set yourself up to be a contributor, so that you can follow our workflow processes:
https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started
https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211773/Content-workflow

**Comment by AbAgrawal15** [comment]
Got the approval for the changes we can go ahead with the merge

**Review by Lauragra — APPROVED** [review]

**Pull request merged**